src/Adapter/Order/CommandHandler/AddProductToOrderHandler.php

```diff
@@ -344,79 +345,15 @@ private function addProductToCart(Cart $cart, Product $product, $combination, $q
         }
     }
 
-    /**
-     * @param AddProductToOrderCommand $command
-     * @param Order $order
-     * @param Cart $cart
-     * @param array $products
-     *
-     * @return OrderInvoice|null
-     */
-    private function createNewOrEditExistingInvoice(
```

Comment:
The update of both created and updated invoice was redundant, since the OrderAmountUpdater manages it all much better So a lot of code can be cleared here:
- no more computations for new invoice
- no more `updateExistingInvoice` method
- therefore even `createNewOrEditExistingInvoice` becomes useless and is managed directly in the `handle` method which makes it simpler to understand